_topic_map.yml

```diff
@@ -231,7 +231,7 @@ Topics:
         File: prerequisites
       - Name: Host Preparation
         File: host_preparation
-      - Name: Containerized Components
+      - Name: Containerized Installation
```

Comment:
@bfallonf As you mention in the BZ, there's not a ton of difference when actually performing a Quick or Advanced install for containerized vs RPM-based. I worry that renaming the topic to "Containerized Installation" suggests that this is now a topic that is going to show you how to specifically kick off a "containerized install", when really other than the private registry edge case mentioned under "Required Images" (about those Ansible / environment variables), most of the topic is a discussion about the differences in how these OCP components behave when containerized vs RPM-based.